games/devtest/mods/unittests/async_env.lua

```diff
@@ -207,3 +207,37 @@ local function test_vector_preserve(cb)
 	end, {vec})
 end
 unittests.register("test_async_vector", test_vector_preserve, {async=true})
+
+local function fill_async()
+	local capacity = core.get_async_threading_capacity()
+	for _ = 1, capacity do
+		core.handle_async(function()
+			local t = os.clock()
+			while os.clock() < t + 0.1 do
+			end
```

Comment:
this returns cpu time not wall time, I'd rather use `get_us_time`
(this can be made race-free if my IPC PR is merged)

Reply:
Done. I assume 10ms is sufficient?

Reply:
Done.